### PR TITLE
Proof IMU_MODE_SWITCHING_ROUTINES #213

### PR DESCRIPTION
**Pull request by dwiyatci**
🕵🏻 Proofread #213.

### CODE REVIEW

File: Comanche055/IMU_MODE_SWITCHING_ROUTINES.agc [hunk 1]
```diff
@@ -341,10 +342,10 @@ IMUFINED	TC	CAGETEST	# SEE THAT NO ONE HAS CAGED THE IMU.
 		TCF	ENDIMU
 
 # Page 1428
-IFAILOK		TC	CAGETSTQ	# ENABLE IMU FAIL UNLESS IMU BEING CAGED.
+IFAILOK		TC	CAGETSTQ	# ENABLE IMU FIAL UNLESS IMU BEING CAGED.
```
Comment (DeeWBee) [inline Comanche055/IMU_MODE_SWITCHING_ROUTINES.agc]: What is the purpose of introducing typos?

File: Comanche055/IMU_MODE_SWITCHING_ROUTINES.agc [hunk 2]
```diff
@@ -57,7 +57,7 @@ SPSCODE		=	BIT9
 		COUNT	07/IMODE
 
 IMUZERO		INHINT			# ROUTINE TO ZERO ICDUS.
-		CS	DSPTAB +11D	# DON'T ZERO CDUS IS IMU IN GIMBAL LOCK AND
+		CS	DSPTAB +11D	# DONT ZERO CDUS IS IMU IN GIMBAL LOCK AND
```
Comment (Kryptos-FR) [inline Comanche055/IMU_MODE_SWITCHING_ROUTINES.agc]: Really?

File: Comanche055/FIXED_FIXED_CONSTANT_POOL.agc [hunk 3]
```diff
@@ -90,7 +91,7 @@ CALLCODE	OCT	00032
 # LOW5		OCT	37		# PINBALL
 # 33DEC		DEC	33		# PINBALL		(OCTAL 41)
 # 34DEC		DEC	34		# PINBALL		(OCTAL 42)
-TBUILDFX	DEC	37		# BUILDUP FOR CONVENIENCE IN DAPTESTING
+TBUILDFX	DEC	37		# BUILDUP FOR CONVIENCE IN DAPTESTING
```
Comment (LeftofZen) [inline Comanche055/FIXED_FIXED_CONSTANT_POOL.agc]: Really

File: Comanche055/IMU_MODE_SWITCHING_ROUTINES.agc [hunk 4]
```diff
@@ -401,12 +402,12 @@ PIPFREE2	TC	IBNKCALL	# ISS WARNING MIGHT COME ON NOW.
...
 		TCF	MODEEXIT
 
-PIPFREE		INHINT			# PROGRAM DONE WITH PIPAS.  DON'T LIGHT
+PIPFREE		INHINT			# PROGRAM DONE WITH PIPAS.  DONT LIGHT
 		CS	IMODES30	# ISS WARNING.
 		MASK	BIT1
 		ADS	IMODES30
 
-		MASK	BIT10		# IF PIP FAIL ON, DO PROG ALARM AND RESET
+		MASK	BIT10		# IF PIP FAIL ON, DO PROG ALSRM AND RESET
```
Comment (DeeWBee) [inline Comanche055/IMU_MODE_SWITCHING_ROUTINES.agc]: Dude. What is with all these mispellings? I see your a Polish national (
Witaj! Mój ojciec jest z Wrocławia); if English isn't your primary language, don't make English corrections.

File: Comanche055/IMU_MODE_SWITCHING_ROUTINES.agc [hunk 5]
```diff
@@ -992,7 +993,7 @@ OCT62		EQUALS	.5SEC		# DEC 50 = OCT 62
 # THE LOW 5 BITS OF THE SCALER READS 10000 FOR THE FIRST INTERVAL AFTER A
 # Page 1447.
 # T1 INCREMENT.  IF SCALPREP DETECTS THIS INTERVAL THE T1,T2 AND SCALER
-# DATA ARE NOT COMPATIBLE AND RETURN IS TO L+1 FOR ANOTHER READING OF THE
+# DATA ARE NOT COMPATABLE AND RETURN IS TO L+1 FOR ANOTHER READING OF THE
```
Comment (DeeWBee) [inline Comanche055/IMU_MODE_SWITCHING_ROUTINES.agc]: INCORRECT SPELLING.

**Comment by Kryptos-FR** [comment]
Unless original code also had those misspelling. In which case we need to weight whether to maintain historical accuracy or just fix some damn spelling mistakes and typos.

**Comment by DeeWBee** [comment]
@Kryptos-FR Personally I'm in favor of fixing spelling mistakes and typos. They don't introduce anything substantive to the material.

File: Comanche055/IMU_MODE_SWITCHING_ROUTINES.agc [hunk 6]
```diff
@@ -1020,16 +1021,16 @@ SCALPREP	EXTEND
 		INCR	MPAC +2		# NO
 		TC	MPAC +2		# YES
 
-# POSTAND RECOVERS TIME AFTER STANDBY.  THE SCALER IS SNAPSHOTTED AND THE
+# POSTAND RECOVERS TIME AFTER STANDBY.THE SCALER IS SNAPSHOTTED AND THE
```
Comment (LeftofZen) [inline Comanche055/IMU_MODE_SWITCHING_ROUTINES.agc]: Incorrect whitespace

File: Comanche055/IMU_MODE_SWITCHING_ROUTINES.agc [hunk 7]
```diff
@@ -1020,16 +1021,16 @@ SCALPREP	EXTEND
...
 # TIME1 TIME2 COUNTER IS SET TO ZERO.  THE LOW 5 BITS OF THE SCALER ARE
-# INSPECTED TO INSURE COMPATIBILITY BETWEEN THE SCALER READING AND THE
+# INSPECTED TO INSURE COMPATABILITY BETWEEN THE SCALER READING AND THE
```
Comment (LeftofZen) [inline Comanche055/IMU_MODE_SWITCHING_ROUTINES.agc]: Incorrect spelling

File: Comanche055/IMU_MODE_SWITCHING_ROUTINES.agc [hunk 8]
```diff
@@ -1040,10 +1041,10 @@ POSTAND		CS	BIT11		# RECOVER TIME AFTER STANDBY
 		TC	POSTAND +3	# T1,T2,SCALER NOT COMPATIBLE
 		EXTEND			# T1,T2 AND SCALER OK
 		DCS	SCALSAVE
-		DAS	MPAC		# FORM DP DIFFERENCE OF POST-STANDBY SCALER
+		DAS	MPAC		# FORM DP DIFFERENCE OF POSTSTANDBY SCALER
```
Comment (LeftofZen) [inline Comanche055/IMU_MODE_SWITCHING_ROUTINES.agc]: Incorrect grammar

**Comment by dwiyatci** [comment]
@DeeWBee @Kryptos-FR @LeftofZen Hold your horses! 🏇 RTFM before being harsh. It was not my decision to make ;) (cc: @chrislgarry). And no, I'm not a Polish national (although my ex is :p). https://emojipedia.org/flag-for-indonesia/

https://github.com/chrislgarry/Apollo-11/blob/master/CONTRIBUTING.md#comments

> Comments in the transcribed code should **match** the scans **exactly**. **This could involve creating a deliberate typo or removing/adding an entire comment**.

**Comment by wopian** [comment]
Sorry @dwiyatci, the repo made the rounds on Twitter and Reddit again so we have the usual wave of people not aware of the contribution process 😢

I should probably get round to adding issue and PR templates to help alleviate some of this, along with a pinned issue regarding reading the contributing guide before reviewing.

**Comment by Kryptos-FR** [comment]
What's the point of keeping the typo? Except making the comment unreadable?
Can we agree to disagree on the CONTRIBUTING.MD?

**Comment by wopian** [comment]
@Kryptos-FR this repository is an archive of the scans. 

The grammar and spelling mistakes were made in 1969 and represent the source code as it was on the near 3,500 pages\[1]\[2] that were scanned in their entirety by Paul Fjeld thanks to the MIT Museum. Changing the comments post-mortem can introduce ambiguous comments that may change the context its referring to (see some of the early pull requests were this was problematic), so this project's goal is to fix the discrepancies introduced from the original digitisation by Paul Fjeld.

\[1]:http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/
\[2]:http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/

**Comment by dwiyatci** [comment]
@wopian Ah, Twitter and Reddit -- and "that usual wave" 🌊😃

Thank you for the clarification btw. Told you we shudda finished this before the 16th of July 2019 hehe 🤭😏🚀

**Comment by chrislgarry** [comment]
@wopian Thats right. Thanks for clarifying.

**Comment by wopian** [comment]
Damn, I'm getting really behind 😓

**Comment by dwiyatci** [comment]
@wopian Can't believe this becomes the last one from my series of PRs this year that you merge :bowtie:

**Pull request merged**